src/components/recentSections/recentSectionRenderStrategy.tsx

```diff
@@ -47,4 +47,14 @@ export class RecentSectionRenderStrategy implements NodeRenderStrategy {
 			onRecentSectionSelected(this.section, OneNoteItemUtils.getAncestry(this.section));
 		}
 	}
+
+	private breadcrumbs(): string {
```

Comment:
Do we need this for normal sections as well?

Reply:
I think that's too much information, because normal sections are always going to belong to a notebook or section group, and the notebook will have path information already. The recent sections are kind of "orphaned", so the path information is useful in that case.